site/es/blog/new-in-devtools-94/index.md

```diff
@@ -0,0 +1,181 @@
+---
+layout: "layouts/blog-post.njk"
+title: "Qué hay de nuevo en DevTools (Chrome 94)"
+authors:
+  - jecelynyeen
+date: 2021-08-24
+updated: 2021-08-24
+description:
+  "Usa DevTools en tu idioma preferido, nuevos dispositivos Nest Hub, nueva insignia de consultas de contenedor CSS y más novedades."
+hero: 'image/dPDCek3EhZgLQPGtEG3y0fTn4v82/GzZ1YTEeHlnhUrbPBzmr.jpg'
+alt: ''
+tags:
+  - new-in-devtools
+  - devtools
+  - chrome-94
+---
+
+*Traducción cedida por [Carlos Caballero](https://carloscaballero.io).*
+
+{% include 'partials/devtools/es/banner.md' %}
+
+
+## Utiliza DevTools en tu idioma preferido {: #localized }
+
+Chrome DevTools soporta más de 80 idiomas, lo que permite que lo puedas utilizar en tu idioma preferido.
```

Comment:
To emphasize this as a new feature, I suggest including the translation for 'now' too
```suggestion
Chrome DevTools ahora soporta más de 80 idiomas, lo que permite que lo puedas utilizar en tu idioma preferido.
```